src/Packages/Starter/database/seeds/UserTableSeeder.php

```diff
@@ -14,12 +14,15 @@ class UserTableSeeder extends Seeder
     public function run()
     {
         $service = app(UserService::class);
-
-        $user = User::create([
-            'name' => 'Admin',
-            'email' => 'admin@admin.com',
-            'password' => bcrypt('admin'),
-        ]);
+        
+        if (!User::where('name', 'admin')->first()) 
```

Comment:
I like following PSR-2 so can we have the bracket on the same line for the if statement.